openquake/server/views.py

```diff
@@ -1227,7 +1231,9 @@ def mmi_tags(request, calc_id):
             content='%s: %s in %s\n%s' %
             (exc.__class__.__name__, exc, 'mmi_tags', tb),
             content_type='text/plain', status=400)
-    return HttpResponse(content=df.to_json(), content_type=JSON, status=200)
+    response_data = {'column_descriptions': EXPOSURE_FIELD_DESCRIPTION,
+                     'exposure_by_mmi': json.loads(df.to_json())}
```

Comment:
It is ugly to convert to json just to call json.loads just to convert again into json. I would try to play with DataFrame.to_dict

Reply:
Done